contracts/asset-proxy/contracts/src/bridges/Eth2DaiBridge.sol

```diff
@@ -0,0 +1,130 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/contracts/src/interfaces/IERC20Token.sol";
+import "@0x/contracts-exchange/contracts/src/interfaces/IWallet.sol";
+import "./ERC20Bridge.sol";
+import "../interfaces/IEth2Dai.sol";
+
+
+// solhint-disable space-after-comma
+contract Eth2DaiBridge is
+    ERC20Bridge,
+    IWallet
+{
+    bytes4 private constant LEGACY_WALLET_MAGIC_VALUE = 0xb0671381;
+    /* Mainnet addresses */
+    address constant public ETH2DAI_ADDRESS = 0x39755357759cE0d7f32dC8dC45414CCa409AE24e;
+    address constant public WETH_ADDRESS = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
```

Comment:
Ideally we don't have to hard code any token addresses. It looks like Eth2Dai supports other token pairs in theory, and those should be able to be used as well. The `fromToken` can probably be passed into the `bridgeData`.